src/connector.js

```diff
@@ -356,46 +368,40 @@ function initTpl() {
 	if ( !suggestionsElement && searchBoxElement && params.unsupportedSuggestions && params.numberOfSuggestions > 0 ) {
 		suggestionsElement = document.createElement( "ul" );
 		suggestionsElement.id = "suggestions";
+		suggestionsElement.role = "listbox";
 		suggestionsElement.classList.add( "rough-experimental", "query-suggestions" );
```

Comment:
`rough-experimental` can be removed
```suggestion
		suggestionsElement.classList.add( "query-suggestions" );
```